src/templates/partials/caddy.hbs

```diff
@@ -1,43 +1,31 @@
 # {{output.header}}
 # {{{output.link}}}
-{{#unless (includes "old" form.config)}}
+{{#unless (minver "2.0.0" form.serverVersion)}}
+# note that Caddy version 1 reached end-of-life > 4 years ago back in 2020
+{{else}}
+{{#if (includes "old" form.config)}}
+# note that Caddy supports only TLSv1.2 and later
+{{else}}
 # note that Caddy automatically configures safe TLS settings
+{{/if}}
 {{/unless}}
 
 # replace example.com with your domain name
-example.com
-{{! This is a big of a kludge due to Caddy restrictions on TLS versions and cipher suites }}
-{{#if (includes "old" form.config)}}
-
-tls {
-    protocols tls1.0 tls1.3
-    {{#if (minver "2.0.0" form.serverVersion)}}
-    ciphers {{{join output.ciphers " "}}}
-    {{ else }}
-    ciphers ECDHE-ECDSA-AES128-GCM-SHA256 ECDHE-RSA-AES128-GCM-SHA256 ECDHE-ECDSA-AES256-GCM-SHA384 ECDHE-RSA-AES256-GCM-SHA384 ECDHE-ECDSA-WITH-CHACHA20-POLY1305 ECDHE-RSA-WITH-CHACHA20-POLY1305 ECDHE-ECDSA-AES128-CBC-SHA ECDHE-RSA-AES256-CBC-SHA ECDHE-RSA-AES128-CBC-SHA ECDHE-ECDSA-AES256-CBC-SHA RSA-AES128-CBC-SHA RSA-AES256-CBC-SHA RSA-3DES-EDE-CBC-SHA
-    {{/if}}
-}
-{{/if}}
-{{#if (includes "intermediate" form.config)}}
+example.com {
 
-# Due to a lack of DHE support, you -must- use an ECDSA cert to support IE 11 on Windows 7
-tls {
-    protocols tls1.2 tls1.3
```

Comment:
I think I will add these commented out, combined with the "Caddy automatically configures safe TLS settings" note, to show what is the actual default, and that it just doesn't need setting.

Reply:
See #262